src/Storefront/Resources/views/storefront/component/shipping/shipping-fields.html.twig

```diff
@@ -6,24 +7,27 @@
 {% block component_shipping_methods %}
     <div class="shipping-methods">
         {% block component_shipping_method %}
-            {% for shipping in page.shippingMethods[:visibleShippingMethodsLimit] %}
+            {% for shipping in feature('ACCESSIBILITY_TWEAKS') ? page.shippingMethods : page.shippingMethods[:visibleShippingMethodsLimit] %}
                 {% sw_include '@Storefront/storefront/component/shipping/shipping-method.html.twig' %}
             {% endfor %}
 
+            {# @deprecated tag:v6.7.0 - block will be removed, all shipping methods will be shown for accessibility #}
             {% block component_shipping_method_collapse %}
-                {% if page.shippingMethods|length > visibleShippingMethodsLimit and visibleShippingMethodsLimit is not same as(null) %}
+                {% if page.shippingMethods|length > visibleShippingMethodsLimit and visibleShippingMethodsLimit is not same as(null) and not feature('ACCESSIBILITY_TWEAKS') %}
                     <div class="collapse">
                         {% for shipping in page.shippingMethods[visibleShippingMethodsLimit:] %}
                             {% sw_include '@Storefront/storefront/component/shipping/shipping-method.html.twig' %}
                         {% endfor %}
                     </div>
 
+                    {# @deprecated tag:v6.7.0 - block will be removed, all shipping methods will be shown for accessibility #}
                     {% block component_shipping_method_collapse_trigger %}
                         <button class="btn btn-link confirm-checkout-collapse-trigger"
                                 type="button"
                                 data-collapse-checkout-confirm-methods="true"
```

Comment:
What about this plugin on `data-collapse-checkout-confirm-methods`?

When this is not used anywhere else I think we can also deprecate the JS plugin `CollapseCheckoutConfirmMethodsPlugin`